plugins/woocommerce-blocks/assets/js/editor-components/upgrade-downgrade-notice/UpgradeDowngradeNotice.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { clsx } from 'clsx';
+
+/**
+ * Internal dependencies
+ */
+import type { UpgradeDowngradeNoticeProps } from './types';
+import './style.scss';
+
+export function UpgradeDowngradeNotice( {
+	children,
+	className,
+	actionLabel,
+	onActionClick,
+	...props
+}: UpgradeDowngradeNoticeProps ) {
+	return (
+		<Notice
+			{ ...props }
+			className={ clsx( 'wc-block-upgrade-downgrade-notice', className ) }
```

Comment:
We should probably [prefix](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/contributors/coding-guidelines.md#naming) this class with `.wc-block-editor-components-`.
```suggestion
			className={ clsx( 'wc-block-editor-components-upgrade-downgrade-notice', className ) }
```

Reply:
Done -> [8ac0fc6](https://github.com/woocommerce/woocommerce/pull/54483/commits/8ac0fc66973d12a8a5e3e8b9cf97f9cd29352b91)